packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -653,6 +679,7 @@ Future<T?> showModalBottomSheet<T>({
   double? elevation,
   ShapeBorder? shape,
   Clip? clipBehavior,
+  BoxConstraints? constraints,
```

Comment:
The API doc for the show methods should explain this parameter. Hopefully a doc macro can be used...

Reply:
I did think about that, but the rest of these params are not documented either, so I thought they might not need it 😏. Would a pointer to the BottomSheet docs on these be enough? Otherwise I can try to use macros, but they probably won't work the same way, as they are params to a function and not field docs.

Reply:
Ok, I somehow missed the references to the other params. I have just added 'constraints' to that list and added a pointer to the `BottomSheet` docs for more details.